### DISCUSSION TITLE
Can't build with bevy_dylib anymore, after upgrade to bevy 0.14

### DISCUSSION THREAD

**Discussion by mlp1802** [bevyengine/bevy#14403] [General]
Dynamic linking is not added as a feature in cargo (as is best practice, I read).
Did cargo clean, cargo cache -a, and removed the target folder.
cargo run --features "bevy/dynamic_linking" 

results in:

error: linking with `g++` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/home/mikkel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/mikkel/.nvm/versions/node/v21.5.0/bin:/home/mikkel/.emacs.d/bin:/home/mikkel/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/mikkel/.local/share/JetBrains/Toolbox/scripts" VSLANG="1033" "g++" "-Wl,--version-script=/tmp/rustc5CzxOe/list" "-Wl,--no-undefined-version" "-m64" "/tmp/rustc5CzxOe/symbols.o" "/home/mikkel/src/ob3d/rust/target/debug/deps/bevy_dylib-ebd50478b30f6287.bevy_dylib.2da87fad2a66e6ab-cgu.0.rcgu.o" "/home/mikkel/src/ob3d/rust/target/debug/deps/bevy_dylib-ebd50478b30f6287.2zopxpqedvhnuggw.rcgu.rmeta" "-Wl,--as-needed" "-L" "/home/mikkel/src/ob3d/rust/target/debug/deps" "-L" "/home/mikkel/src/ob3d/rust/target/debug/build/blake3-260d91b263007cb8/out" "-L" "/home/mikkel/src/ob3d/rust/target/debug/build/blake3-260d91b263007cb8/out" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/home/mikkel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_internal-7ebc440219e05f2f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_winit-563aa5c732fbad2f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libaccesskit_winit-d4711a7955dc9eaf.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libwinit-c7fdbfc34705887a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libx11rb-a72377dd5bded469.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libas_raw_xcb_connection-31c66382c64e9f57.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libx11rb_protocol-ca3e800769fa64a3.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcalloop-94ee39f32c18f8ff.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpolling-921abe3439163318.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librustix-f082adb2873a73e1.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblinux_raw_sys-ac9a1a8112a881a9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libx11_dl-4c8cf4829b524fb1.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libxkbcommon_dl-e7388bcb3ae05a1e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libxkeysym-4a1ee271cce1e687.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libdlib-f4e7a7beba59e6d0.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcursor_icon-212159284c63ba5c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libdpi-b957a35133b6eabb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_ui-7b34bda92c051b62.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtaffy-0bcc5fe60cd16bb6.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libslotmap-830e1257728c87e8.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgrid-ba237bb3997a1d2a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_text-fae2ac347623f28c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libglyph_brush_layout-e4ef4d27f72a586f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libxi_unicode-6fe75f574d1b1c17.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libapprox-7fc52cea1e18d8da.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libab_glyph-8117345783f49711.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libowned_ttf_parser-ca4cb4b56b1ad8fb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libttf_parser-f970305852cca7df.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libab_glyph_rasterizer-c7ff7d75bfc5a99b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_log-02379893eb8f61e6.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtracing_subscriber-eebbe0be674af716.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnu_ansi_term-93677ef03b08ce11.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liboverload-b5e5397f751d766d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libsharded_slab-873e69e323bda4c5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libmatchers-f784d297f9be416e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libregex_automata-99ed384155c05c39.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libregex_syntax-0b1edbc7b1498d6d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtracing_log-76f0fba707735a47.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_gltf-43020080701d05a9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpercent_encoding-58ec43dc2b1b335c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbase64-9d373da04e43ec3b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgltf-9cf9344bf2b5df16.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgltf_json-ea418bc31a529ec9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libserde_json-5f6da879221942a9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libitoa-3638fb367f996114.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libryu-90c4cf4e6410ee6b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblazy_static-e63c79e5761eeff5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_scene-78b15a7a6e4d8d05.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_gizmos-9fd2f7f3b003d357.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_sprite-dd51102114ab454a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librectangle_pack-6545e0358f57b701.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libguillotiere-3ede5a902dd42df8.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libsvg_fmt-6cb801289f15bb43.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libeuclid-66e67efe63563c94.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_pbr-05730b5dc4c8d19d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_gilrs-815cfce5e94f8ba5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgilrs-1471885faf37916a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfnv-847f99faea182ee5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgilrs_core-54b73f792643f5b7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblibudev_sys-3ad43ef61869a532.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libinotify-fe5e58ea39ec358a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libinotify_sys-a274ca0e756ed26a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libvec_map-7df91e5fb223fe09.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnix-17bdb4f2cbf3d7a9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_input-5b8acd7e1113f289.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_core_pipeline-0d4eefe27518b36d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libradsort-c9a365295568c76e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_animation-f0d2beb565c9a438.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_render-687d833074e8327c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libruzstd-f52fdd0662abd3be.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtwox_hash-fe7871b726fc2a49.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnaga_oil-5eddf1719da88abf.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libdata_encoding-d070ee149620f1fb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libunicode_ident-4c48c49fe5465250.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libregex-1287d76f9013f19a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libregex_automata-4789e385c4fee338.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libaho_corasick-a00fc6d56563f651.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libmemchr-12ee59e3c222cb6f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libregex_syntax-7acda0b66e7fd5be.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_mikktspace-65fe7cb372e417ba.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libktx2-1adf8af5980709ee.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libimage-86cca2dbb3c36946.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbyteorder-e40610a39e556e87.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpng-3cf12c995c5a2946.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libflate2-0d7fc7ef0793da0e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libminiz_oxide-f5d532d3ccb06ea4.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfdeflate-26901b654d4783ca.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libsimd_adler32-931c3e21f69b3434.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcrc32fast-c67b430b3edde48e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbitflags-5df622ceebb82bc9.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libzune_jpeg-313f110bdeb13f4f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libzune_core-0199519878f77b35.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnum_traits-6e6b51402cde9c61.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libhexasphere-482eacb2e11c0d0d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconstgebra-99b738a90ed41b13.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconst_soft_float-bda4000faf362f30.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_diagnostic-c289694932ebe21d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconst_fnv1a_hash-fb9a151b3f2d1102.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_window-e741d6c2e0efb5e5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libwgpu-6b4fd8eee7630263.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libstatic_assertions-bcf810a07c8ea391.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libwgpu_core-d6e1c7549b1b7e66.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libwgpu_hal-585d0501c9fb8247.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librenderdoc_sys-4f54096648ba0e52.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgpu_descriptor-5f1b04d595818c0c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgpu_descriptor_types-c6c72860a7f1f432.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libkhronos_egl-65b51a044734f94d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblibloading-30813b7d49cd5132.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libprofiling-78e76e6bab00bc98.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgpu_alloc-323769e6ac6b3b53.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgpu_alloc_types-56325ed20eab49f7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libash-4ceb369973c83422.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblibloading-93796250a004e774.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnaga-fd842a9857178f01.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librustc_hash-f7ed409781f2eedb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libhexf_parse-84a3b2050d72dc4d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libunicode_xid-c08e1988890d4d70.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblog-ad2817de62b72c31.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcodespan_reporting-889fb168c25ef7af.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libunicode_width-e4f125ef96f24141.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtermcolor-1188fda05dbe2000.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libspirv-c43bff7eaf7d73c3.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbit_set-78b46f1581b409bd.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbit_vec-b21bf3bc6299d53d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libraw_window_handle-b0320b18135753d3.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libglow-3093444d62919bc7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_time-6ca96b685d690c43.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_asset-4bf29a3c4f3b1634.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libblake3-8bb9f91863c62563.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconstant_time_eq-910a02c030f6abe7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libarrayref-d06b21e2be4206f5.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_broadcast-89a01b78ffdd77fc.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libevent_listener-838628e42facaa55.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libron-9614786c5918d2e8.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbase64-ed712d85a3ea59ba.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libparking_lot-8cb6491c61e8da74.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libparking_lot_core-2a056c233a02eb1c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblock_api-6f377c9a3f3a2276.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libscopeguard-a94479ba6da262ef.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcrossbeam_channel-78ae2da35a4f968a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_fs-b361d59e181ec4a4.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libblocking-1795545674505b2b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpiper-ace3e19788b4f583.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libatomic_waker-c8248d00f669574a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_lock-fa3201f0dc7785d7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_transform-3529a611c507de7d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_hierarchy-6f1e28c307c18991.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_core-9881556d8911fd27.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_color-1addc3a88674b331.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libencase-cc84f5554acb7d83.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconst_panic-f1c847808672051f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libwgpu_types-623384cfa5ae932d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_math-905577a5ab612b5a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_a11y-b8002df45184b43b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libaccesskit-af923c0a0c832bca.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_app-a77d0fd66b416089.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_ecs-06341eb55fc986c1.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbitflags-862e961f882607d0.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libnonmax-c1ebd4a18f695561.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_tasks-dd040ed323aa4339.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_channel-aaffaa7b94d5df07.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libevent_listener_strategy-cbce0214b3aee9df.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libevent_listener-fedda93074f9f863.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_executor-f9bd27816058df3e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libslab-240ca553b14a6ff7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfutures_lite-9888b2875685c0c2.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfutures_io-1665dce99ff93865.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfutures_core-49c6f4eab145f677.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libparking-c910b7f76126c12f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfastrand-0607dd522fac2729.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libconcurrent_queue-14afe892be0ea3bc.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcrossbeam_utils-1427c161da6f7ebb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libasync_task-ed736b6790aa420d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libarrayvec-76d6f09483381982.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfixedbitset-d2542f03f6e612fd.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_reflect-798ea0ababaf8b08.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libuuid-09d17fa7d4f2c2ec.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libsmol_str-cba404ef0e14c3ee.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpetgraph-d9b409531a1d342a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libindexmap-38cc190e0ed90b9b.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libequivalent-4c66b6e292ca92bb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libfixedbitset-a064352c3a1f8839.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liberased_serde-6276e270c5ab1a6f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtypeid-be27154a0b82da94.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libsmallvec-7f9f9f668d34313d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libglam-f34744b3e99179cb.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librand-edf803be3eaf68f4.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librand_chacha-1817b152583752e7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libppv_lite86-10df8adb3efb00c4.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/librand_core-be547dfbc41d582d.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbytemuck-ef705c26f5bc053c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libdowncast_rs-beb7884c53ec7e5e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libthiserror-37bcdae87ecc833a.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_utils-29850c69c05a29a3.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libweb_time-bb00d9f057d9da01.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtracing-4cd1cdb48e2ae073.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libpin_project_lite-1fbde0a9251e5faa.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libtracing_core-93db08c89d17290e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libhashbrown-b91e3474d7f68076.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libserde-903888e576c32025.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liballocator_api2-4234d1216d3a925e.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libahash-b90bd77b4b9d704f.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libgetrandom-101179d22a79d539.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/liblibc-c536b89bf0398103.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libzerocopy-362c784514719b71.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libthread_local-fdd2ad41199b7fc7.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libcfg_if-218aa1763481c62c.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libonce_cell-a612189cf1ea4533.rlib" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_ptr-43001efa5705aee8.rlib" "-L" "/home/mikkel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bdynamic" "-lstd-d5189b81a4fa4d36" "-Wl,-Bstatic" "/home/mikkel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-d9076ee5964191bf.rlib" "-Wl,-Bdynamic" "-ldl" "-ludev" "-ldl" "-ldl" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/home/mikkel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/mikkel/src/ob3d/rust/target/debug/deps/libbevy_dylib-ebd50478b30f6287.so" "-shared" "-Wl,-z,relro,-z,now" "-Wl,-O1" "-nodefaultlibs" "-fuse-ld=lld"
  = note: ld.lld: error: version script assignment of 'global' to symbol '__rust_alloc' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_alloc_error_handler' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_alloc_error_handler_should_panic' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_alloc_zeroed' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_dealloc' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_no_alloc_shim_is_unstable' failed: symbol not defined
          ld.lld: error: version script assignment of 'global' to symbol '__rust_realloc' failed: symbol not defined
          collect2: error: ld returned 1 exit status
          

error: could not compile `bevy_dylib` (lib) due to 1 previous error

**Comment by alice-i-cecile** [comment]
Presumably you mean Bevy 0.14 in the title, right?

**Reply by mlp1802** [reply]
Yes sorry, changing it now
